src/table/cells/handle-key-press.js

```diff
@@ -67,60 +57,79 @@ export const preventDefaultBehavior = (evt) => {
   evt.preventDefault();
 };
 
-export const moveFocus = (evt, rootElement, rowIndex, colIndex, selState) => {
+export const moveFocus = (evt, rootElement, cellCoord, setFocusedCell, selState) => {
   preventDefaultBehavior(evt);
   removeCurrentFocus(evt);
   const rowAndColumnCount = getRowAndColumnCount(rootElement);
-  const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex, selState);
-  moveToNextFocus(rowAndColumnCount.rowElements, nextRow, nextCol);
+  const nextCellCoord = arrowKeysNavigation(evt, rowAndColumnCount, cellCoord, selState);
+  focusCell(rowAndColumnCount.rowElements, nextCellCoord);
+  setFocusedCell(nextCellCoord[1]);
```

Comment:
to set number instead of an array to fix performance issue

Reply:
but this is only the row number, how would that work?

Reply:
it is the nextCol, the column number.

Reply:
yes, 0 is the first... of course hehe. But it still missing the row index in that case

Reply:
yes, to fix the performance issue we talked about yesterday, here only column number is set, then rest the tab index of all cells in this column no matter which row it is.

Reply:
What happens if you have a lot of columns and move fast to the right?

Reply:
good point. it is a bit laggy. but compared to move up and down, it is less common to happen. Currently, I do not come up with a way of not using setFocusedCell.